### PR TITLE
Filter and upload functions working

### PR DESCRIPTION
**Pull request by cartershadd**
Filter and upload functions working, as does the delete function and login/logout options.

### CODE REVIEW

File: src/Pages/ArtCollection.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+
+class ArtCollection extends Component {
```
Comment (nburn42) [inline src/Pages/ArtCollection.js]: ?

File: src/Pages/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    // Make sure we un-register Firebase observers when the component unmounts.
+    componentWillUnmount() {
+        this.unregisterAuthObserver();
+    }
+
+    render() {
+        return (
+            <div>
+
+                    <Navbar collapseOnSelect expand="*" bg="dark" variant="dark">
```
Comment (nburn42) [inline src/Pages/Navigation.js]: Formatting is weird

File: src/Pages/PersonalUploads.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        this.state = {
+            paintings: [],
+        };
+    }
+
+    deletePainting = (id) => {
+        db.collection("paintings").doc(id).delete().then(function() {
+            console.log("Document removed successfully!");
+            // eslint-disable-next-line no-lone-blocks
+            {
```
Comment (nburn42) [inline src/Pages/PersonalUploads.js]: You don't need this bracket

File: src/Pages/uploadPage.js [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            liquitex: false,
+            other: false,
+            blowtorch: false,
+            image: '',
+            files: []
+        };
+        console.log(firebase.auth())
+    }
+
+    onSubmit = (e) => {
```
Comment (nburn42) [inline src/Pages/uploadPage.js:36]: Add a check to make sure the user is signed in

File: src/Pages/uploadPage.js [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+    };
+
+    render() {
+        return (
+            <Form className="upload-form" onSubmit={this.onSubmit}>
+                <Form.Label className="form-label" column={1}>Upload Your Work Here!</Form.Label>
+                <FilePond
+                    ref={ref => (this.pond = ref)}
+                    files={this.state.files}
+                    allowMultiple={true}
```
Comment (nburn42) [inline src/Pages/uploadPage.js:155]: Currently your upload only saves a single file

**Review by nburn42 — APPROVED** [review]

File: src/Pages/ArtCollection.js (hunk 1 above)
Comment (cartershadd) [reply src/Pages/ArtCollection.js]: deleted now, it was a component i made that i never ended up using.

File: src/Pages/Navigation.js (hunk 2 above)
Comment (cartershadd) [reply src/Pages/Navigation.js]: fixed it

File: src/Pages/PersonalUploads.js (hunk 3 above)
Comment (cartershadd) [reply src/Pages/PersonalUploads.js]: ok, bracket and messaged removed.

File: src/Pages/uploadPage.js (hunk 5 above)
Comment (cartershadd) [reply src/Pages/uploadPage.js:155]: ok, i want to leave it that way.

File: src/Pages/PersonalUploads.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+            }
+        }).catch(function(error) {
+            console.error("Error removing document: ", + id, error);
+        })
+    };
+
+    componentDidMount() {
+        firebase.auth().onAuthStateChanged(
+            (user) => {
+                this.setState({isSignedIn: !!user});
```
Comment (nburn42) [inline src/Pages/PersonalUploads.js:28]: Add is signed in to state in the constructor as false

File: src/Pages/uploadPage.js (hunk 4 above)
Comment (nburn42) [reply src/Pages/uploadPage.js:36]: You can use the same is signed in code from personal upload page

**Pull request merged**